scripts/cache_models.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+from allennlp.commands import DEFAULT_MODELS
+from allennlp.common.file_utils import cached_path
+
+value = os.environ.get('CACHE_MODELS', 'false')
+if value.lower() == "true":
+    models = DEFAULT_MODELS.items()
+    print("CACHE_MODELS is '%s'.  Downloading %i models." % (value, len(models)))
+    for i, (model, url) in enumerate(models):
+        print("Downloading '%s' model from %s" % (model, url))
+        print("Saved at %s" % cached_path(url))
+else:
+    print("CACHE_MODELS is '%s'.  Not caching models." % value)
```

Comment:
I mean, this is sort of misleading, since you're printing "false" even if it's not set. I'm not sure we care though.

Reply:
@joelgrus, if it's not set then I print `CACHE_MODELS is ''`.